src/app/cases/case-filters/case-filters.component.ts

```diff
@@ -75,15 +75,22 @@ export class CaseFiltersComponent implements OnInit {
     });
   }
 
-  resetFilters(): void {
-    this.initFiltersForm();
-    this.filtersToArray();
-  }
   onFormChange(): void {
     this.filtersToArray();
   }
 
   ngOnInit(): void {
     this.initFiltersForm();
+    this.subscriptions.push(
+      this.filterService.getFilters().subscribe((response: any) => {
+        if (!response.filters.length) {
```

Comment:
if there is a child - parent or parent - child relationship between the components, it is not the best practice to use services. I have created a PR that we will not merge, to show you a possible solution and to use the parent -> child comunication. It is done for samples and if you think it is ok like this, you will have to make it the same for the rest

https://github.com/hzi-braunschweig/SORMAS-Angular/pull/134